src/main/java/codeu/model/data/User.java

```diff
@@ -32,11 +34,16 @@
    * @param passwordHash the password hash of this User
    * @param creation the creation time of this User
    */
-  public User(UUID id, String name, String passwordHash, Instant creation) {
+  public User(UUID id, String name, String passwordHash, Instant creation, String description) {
     this.id = id;
     this.name = name;
     this.passwordHash = passwordHash;
     this.creation = creation;
+    this.description = description;
+  }
+
+  public User(UUID id, String name, String passwordHash, Instant creation) {
```

Comment:
Lets add documentation for this constructor as well.

Reply:
Got it.